server/translations.ts

```diff
@@ -141,22 +144,52 @@ function getDocument(filePath) {
     };
   }
 
+  function getCommitBehindFromLatest(filename, commitHash) {
+    if (commitHashCache[filename] === undefined) {
+      try {
+        commitHashCache[filename] = execSync(
+          `git rev-list --count ${commitHash}..HEAD -- ${filename}`,
```

Comment:
I was able to reduce 930 seconds to 214 seconds with `git rev-list --count ${commitHash}..HEAD -- ${filename}` command for ja locale files. 

- AS IS
  - 6840 files (with source commit 1825 files): takes 930 seconds. (15m 30s)
- TO BE
  - 6840 files (with source commit 1825 files): takes 214 seconds. (3m 34s)

Reply:
But still we have to consider up to 10000 files. 

Since people are usually mostly interested in having a high lagged count of sourcecommits, information about how far behind sourcecommit is, I don't think it's necessary to keep it real-time.

So I don't think it's a bad idea to store information about how far behind a sourcecommit is in a file, and update it with git actions every week.